R/mutate.R

```diff
@@ -304,35 +301,20 @@ rename_vars <- function(data, from, to) {
 #' @examples
 #' collapsed <- collapse_cat(iris,
 #'     var = "Species",
-#'     list(S = "setosa", V = c("versicolor", "virginica"))
+#'     c("versicolor", "virginica"),
+#'     levels_to = "V"
 #' )
 #' cat(code(collapsed))
-#' head(collapsed)
+#' tail(collapsed)
 #'
 #' @author Stephen Su
 #' @export
-collapse_cat <- function(data, var, collapsed = NULL, name = NULL) {
+collapse_cat <- function(data, var, levels_from, levels_to, name = NULL) {
```

Comment:
I think this runs into the same issues we discussed for `rename_vars()` - a single, named list is easier to work with (it's *obvious* which levels are being renamed to what).